cli/azd/.vscode/cspell-azd-dictionary.txt

```diff
@@ -51,11 +51,13 @@ azureyaml
 Backticks
 bicepparam
 BOOLSLICE
+bubbletea
```

Comment:
@chlowell used Bubble Tea here because he didn't notice our existing picker technology - I'm going to move what he has over to our picker so we don't have to opt into bubble tea yet.